OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -2265,6 +2285,28 @@ internal static void UpgradeActorRules(int engineVersion, ref List<MiniYamlNode>
 						node.Key = "Aircraft";
 						node.Value.Nodes.Add(new MiniYamlNode("CanHover", "True"));
 					}
+
+					var mplane = node.Value.Nodes.FirstOrDefault(n => n.Key == "-Plane");
+					if (mplane != null)
+					{
+						// Check if a Helicopter trait was renamed to Aircraft
+						// In that case, we don't want to straight negate it with -Aircraft again
```

Comment:
I'm not sure what problem you're trying to solve here. An actor that simultaneously defines both `Helicopter` and `-Helicopter`? That doesn't make much sense. Also the comment refers to helicopters while it's checking for `Plane` :p. Additionally, this code will only be able to run when `depth` is 0, otherwise `mplane`/`mheli` will be null, so this code will always run before the other two checks (which only run at depth 1), so a renaming cannot have happened at this point.

Why not just add the two rules from the original PR, just with the traits inverted?